app/livechat/client/views/app/tabbar/visitorForward.js

```diff
@@ -94,3 +121,66 @@ Template.visitorForward.events({
 		this.cancel();
 	},
 });
+
+Meteor.startup(function() {
+	Tracker.autorun(function() {
+		if (Meteor.userId()) {
+			Notifications.onUser('forward', (type, data) => {
+				const user = Meteor.users.findOne(data.transferData.userLoggedIn);
+				switch (type) {
+
+					case 'handshake':
+
+						modal.open({
+							title: TAPi18n.__('LiveChat'),
+							text: TAPi18n.__('Username_wants_to_forward_livechat_Do_you_want_to_accept', { username: user.username }),
+							html: true,
+							showCancelButton: true,
+							allowOutsideClick: false,
+							confirmButtonText: TAPi18n.__('Yes'),
+							cancelButtonText: TAPi18n.__('No'),
+						}, (isConfirm) => {
+							if (isConfirm) {
+								Meteor.call('livechat:transfer', data.transferData, (error, result) => {
+									if (error) {
+										toastr.error(t(error.error));
+									} else if (result) {
+										toastr.success(t('Transferred'));
+										Notifications.notifyUser(data.transferData.userLoggedIn, 'forward', 'transferred', { transferData: data.transferData });
+										FlowRouter.go(`/live/${ data.roomId }`);
+									} else {
+										toastr.warning(t('No_available_agents_to_transfer'));
+									}
+								});
+							}
+						}, () => {
+							Notifications.notifyUser(data.transferData.userLoggedIn, 'forward', 'deny', { transferData: data.transferData });
+						});
+
+						Meteor.setTimeout(() => {
+							modal.close();
+						}, data.transferData.timeoutAgent);
+
+						break;
+
+					case 'deny':
+						const userdeny = Meteor.users.findOne(data.transferData.userId);
+						Meteor.clearTimeout(data.transferData.timeout);
+						modal.open({
+							title: TAPi18n.__('LiveChat'),
+							text: TAPi18n.__('Username_denied_the_OTR_session', { username: userdeny.username }),
```

Comment:
the `Username_denied_the_OTR_session` is not the correct message for this feature.